requirements.txt

```diff
@@ -1,8 +1,7 @@
-PyPrint~=0.2.5
+PyPrint~=0.2.6
 Pygments~=2.1
-setuptools>=19.2
 # Do *not* use 0.3 which is not backwards compatible to common clang versions
-libclang-py3~=0.2.0
```

Comment:
IMO this shouldnt be changed.  See https://github.com/coala/coala/issues/3313#issuecomment-267873619